lib/mixlib/install/generator/base.rb

```diff
@@ -47,6 +47,7 @@ def self.get_script(name, context = {})
           if File.exist? "#{script_path}.erb"
             # Default values to use incase they are not set in the context
             context[:base_url] ||= "https://omnitruck.chef.io"
+            context[:headers] ||= options.user_agent_headers
```

Comment:
I'm working on creating a string version of this option so we don't have to use `join` all over the codebase.